web-extension/_locales/de/messages.json

```
@@ -11,6 +11,10 @@
     "message": "neuen Eintrag erstellen",
     "description": "Text of create new entry button"
   },
+  "createAnotherEntryButtonText": {
+    "message": "sie einen weiteren Eintrag erstellen",
```

```suggestion
    "message": "einen weiteren Eintrag erstellen",
```
That would be grammatically better 👍